client/src/includes/initStimulus.ts

```diff
@@ -1,6 +1,29 @@
-import type { Definition } from '@hotwired/stimulus';
+import type { Controller, Definition } from '@hotwired/stimulus';
 import { Application } from '@hotwired/stimulus';
 
+class WagtailApplication extends Application {
+  queryController(identifier: string): Controller | null {
+    return this.getControllerForElementAndIdentifier(
+      document.querySelector(
+        `[${this.schema.controllerAttribute}="${identifier}"]`,
+      )!,
+      identifier,
+    );
+  }
+
+  queryControllerAll(identifier: string): Controller[] {
```

Comment:
Can we please add a basic JSDoc here. 

I love the function though, super helpful. 

We probably should also add this as a requested feature in Stimulus itself.